src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/StartupObjectsEnumProvider.cs

```diff
@@ -84,7 +90,7 @@ public async Task<ICollection<IEnumValue>> GetListedValuesAsync()
             }
 
             IEntryPointFinderService? entryPointFinderService = project.LanguageServices.GetService<IEntryPointFinderService>();
-            IEnumerable<INamedTypeSymbol>? entryPoints = entryPointFinderService?.FindEntryPoints(compilation?.GlobalNamespace, SearchForEntryPointsInFormsOnly);
+            IEnumerable<INamedTypeSymbol>? entryPoints = entryPointFinderService?.FindEntryPoints(compilation?.GlobalNamespace, _searchForEntryPointsInFormsOnly);
```

Comment:
Do the old property pages use the `IEntryPointFinderService`? If so, do they ever pass "true" for the second parameter?

If not, I suggest some manual testing to make sure everything is working as expected when `_searchForEntryPointsInFormsOnly` is "true".

Reply:
I was testing this yesterday, and we used a different method: Interop.IVBEntryPointProvider in [ApplicationPropPageVBWinForms.vb](https://github.com/dotnet/project-system/blob/9b25af877a11bb4011f38b21c067e2e0475c79c4/src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPageVBWinForms.vb#L708). Since we already use the `IEntryPointFinderService` for the new properties pages (at least C# with that flag turned off), I tested what happened when I called it with a true flag; I found out that it does have a bug: https://github.com/dotnet/roslyn/issues/59696.